src/install/ai-monitoring/agent-lang/python-jupyter-config.mdx

```diff
@@ -0,0 +1,32 @@
+---
+componentType: default
+headingText: Update your code for the Python agent
+freshnessValidatedDate: never
+---
+
+1. You want to update your code so that the agent initializes when you spin up your script or notebook. Keep in mind that you need to define your app name for the agent to initialize:
+
+    ```py
+    newrelic.agent.initialize("newrelic.ini")
+    newrelic.agent.register_application(timeout=10)
+    ```
+
+2. Add LLM calls with New Relic annotations to a method in your code:
+
+    ```py
+        bedrock_runtime = boto3.client("bedrock-runtime", "us-east-1")
+
+        runTitan(bedrock_runtime)
```

Comment:
Please rename `runTitan` to `evaluate_llm`. Then remove the rest of these `run` methods.

Reply:
We also want to show a stub implementation of `evaluate_llm`.

```
@newrelic.agent.background_task()
def evaluate_llm():
    # implement calls to your LLM
    pass
```

Reply:
Thinking about it, let's drop the line 17 (`bedrock_runtime =`) and remove that parameter from the method call.